virt/virt-2-4-release-notes.adoc

```diff
@@ -231,3 +231,28 @@ the nodes of any clusters where {VirtProductName} is deployed. The nodes might n
 drain if there are virtual machines running on top of them.
 
 ** The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance[put nodes into maintenance].
+
+* Importing a RHV VM fails if the target VM name exceeds 23 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
```

Comment:
This initialism hasn't been declared in this file yet. 
The notable technical changes section does mention 'Red Hat Virtualization' (but not RHV). Since this is a series of known issues relating to RHV in close proximity, and the previous instance is more isolated, it probably makes more sense to spell it out in full here as 'Red Hat Virtualization (RHV)' and then continue to use RHV but I leave that to your discretion.